concourse/pipelines/replicator.py

```diff
@@ -224,6 +225,7 @@ class DeployStatus(Enum):
     SUCCEEDED = 0
```

Comment:
please convert to IntEnum

also: shift 0 -> 1; 1->2, 2->4, 3->8 (to allow reasonable bitwise operations)